### DISCUSSION TITLE
Upcoming DIPY lab meetings

### DISCUSSION THREAD

**Discussion by arokem** [dipy/dipy#2358] [2. Announcements]
All DIPY lab meetings to be held through [Zoom](https://washington.zoom.us/j/812354748) and are at *the most* 60 minutes long.

- [March 26th, 2019, 1600 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20190326T1600): @arokem will do a practice of DIPY presentation about DKI/HCP cross-validation for the [BRAIN PI meeting](https://www.braininitiative.org/events/pimeeting/).

- [April 9th, 2019, 1600 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20190409T1600): Discussion about seeding strategies, led by @gabknight. We'll discuss #1627, #1648, #1635 and others.

- [April 23rd, 2019, 1600 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20190423T1600): Led by @skoudoro. Topic TBD (probably something code-oriented :smile:).

- [May 7th, 2019, 1600 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20190507T1600): @RafaelNH and/or @mvgolub will lead a discussion about https://github.com/nipy/dipy/pull/1744

- [May 21st, 2019, 1600 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20190521T1600): Prep for OHBM (practice presentations round robin, bring your own poster/talk).

**Comment by jchoude** [comment]
Looking nice. Is there a way to send reminders, or we should watch this issue, or something else?

**Comment by arokem** [comment]
Be sure to watch this issue. I will use this to send reminders. I'll also send a reminder email to the dipy@python.org mailing list a few days before each meeting.

**Comment by jchoude** [comment]
Perfect thanks.

**Comment by arokem** [comment]
We'll have the first of our series of online DIPY lab meetings planned for this spring tomorrow (March 26th) at 1600 GMT (9 AM PST/noon EST). 

To join the meeting, please use the following Zoom link: https://washington.zoom.us/j/812354748. You will need to download the Zoom app to join, so please join a few minutes early, if you have not downloaded the app yet. 

The agenda is for me to present some cloud-based computations using [cloudknot](https://richford.github.io/cloudknot/). 

We'll also spend a few minutes discussing PRs/Issues that are in progress.

**Comment by arokem** [comment]
Reminder: tomorrow (April 8th) at 1600 UTC (find your local time [here](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20190409T1600)) we will have our second DIPY lab meeting. We'll have @gabknight lead the discussion. 

To join, please follow the link: https://washington.zoom.us/j/812354748

**Comment by gabknight** [comment]
Topic suggestion for a future meeting: DOI for DIPY versions #1725 ? + Versioning strategy: number, name, frequency ?

**Comment by mdesco** [comment]
If I may suggest, I would like to see a meeting on future dev and "vision"
of Dipy.

I was impressed by everything new in Dipy during the workshop but the field
is so wide and vast, I would be interested in a discussion about
priorities, focus, etc...

Max

**Comment by arokem** [comment]
@mdesco : We did have some discussion of this on the Friday of the DIPY workshop. We were also talking today about devoting some time to priority setting and long-term vision on our next lab meeting (two weeks from today). Would be great if you (and folks from your lab who are interested) could join!

**Comment by arokem** [comment]
Reminder: tomorrow (April 23rd) at 1600 UTC (find your local time [here](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20190423T1600)) we will have a DIPY lab meeting. @skoudoro is up to give an update on recent developments, and we can spend some time discussing the roadmap to 1.0

To join, please follow the link: https://washington.zoom.us/j/812354748

**Comment by arokem** [comment]
Some notes are here: https://docs.google.com/document/d/1VKSFEiY-d31n2NFPsp0IYKOBflxTWWLW0QWsbYXZbbc/edit?ts=5cbf3791#

**Comment by RafaelNH** [comment]
Reminder: tomorrow (May 7th) at 1600 UTC (find your local time [here](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20190507T16&p1=1440)), me and @mvgolub will show some recent developments on an algorithm for fitting the free-water DTI model to single shell data (PR #1744).

**Comment by mvgolub** [comment]
Here are some implementation notes taken from my thesis: [fwDTI_notes.pdf](https://github.com/nipy/dipy/files/3148628/fwDTI_notes.pdf), see you at the meeting.

**Comment by arokem** [comment]
As always, this is the link for the call: https://washington.zoom.us/j/812354748

**Comment by arokem** [comment]
Just a reminder that the DIPY virtual lab meeting will take place tomorrow (May 21st) at 1600 UTC. 

The call will take place at: https://washington.zoom.us/j/812354748

The agenda for tomorrow's meeting is to discuss plans for OHBM. 

I would also like to devote a few minutes to discuss how we should go about these lab meetings during the summer.

**Comment by skoudoro** [comment]
:+1:  sounds good!

**Comment by arokem** [comment]
Closed through #1841
